reactNativeProject/src/screens/Main/index.js

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { View } from "react-native";
+
+import { connect } from "react-redux";
+import { actionCreators } from "../../redux/item/actions";
+import Title from "./components/Title";
+import Input from "./components/Input";
+import styles from "./styles";
+import TodoList from "./components/TodoList";
+
+const mapStateToProps = store => ({
```

Comment:
You don´t need this `mapStateToProps` if you´re going to get the items in the `TodoList` anyways